tests/test_main.F90

```diff
@@ -29,13 +29,15 @@ program test_main
    write(*,*) ''
 
    ! Read input file and initialize grid
-   call cc_read_config(Config, GridState, rc)
+   call cc_read_config(Config, GridState, ChemState, rc)
    if (rc /= CC_SUCCESS) then
       errMsg = 'Error reading configuration file: ' // TRIM( configFile )
       call cc_emit_error(errMsg, rc, thisLoc)
       stop 1
    endif
 
+   ! call cc_read_species(Config, rc)
```

Comment:
Should we add a test?

Reply:
This is called during the read_config. Already tested

Reply:
I need to remove that comment

Reply:
There doesn't appear to be any testing added in this PR regarding the correctness of the species info that is read in. Should we add something to that effect?